core/storage/base_model/gae_models_test.py

```diff
@@ -111,6 +111,60 @@ class TestVersionedModel(base_models.VersionedModel):
     SNAPSHOT_CONTENT_CLASS = TestSnapshotContentModel
 
 
+class TestCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
+    @classmethod
+    def _get_instance_id(cls, instance_id, version):  # pylint: disable=unused-argument
```

Comment:
See above re notes.

You can prefix an argument with unused_ to signal to pylint that it's unused, I think.

Reply:
Done, changed the method to actually use it.